src/main/java/com/uber/cadence/internal/worker/ActivityWorker.java

```diff
@@ -157,26 +160,31 @@ private TaskHandlerImpl(ActivityTaskHandler handler) {
 
     @Override
     public void handle(MeasurableActivityTask task) throws Exception {
-      options
-          .getMetricsScope()
+      Scope metricsScope =
+          options
+              .getMetricsScope()
+              .tagged(
+                  ImmutableMap.of(MetricsTag.ACTIVITY_TYPE, task.task.getActivityType().getName()));
```

Comment:
Why is it not using the activity type LoggerTag on the line 176?

Reply:
Metrics and logging tag names are different.